app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitValidator.java

```diff
@@ -38,12 +39,13 @@ public HabitValidator(DisplaysErrorMessages fragment) {
      * @param habitReason the habit reason
      * @param habitDate the habit date
      * @param tracker the tracker for days of the week
+     * @param privacyButtons
      * @return true, if habit is valid, false otherwise
      */
     public boolean isHabitValid(String habitTitle,
                                 String habitReason,
                                 String habitDate,
-                                DaysTracker tracker) {
+                                DaysTracker tracker, PublicPrivateButtons privacyButtons) {
```

Comment:
Habit validator now takes the privacy buttons as a parameter.

Reply:
minor: should go on a new line to match the other parameters